frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -60,16 +64,16 @@ export const TopologyPage: React.FC<TopologyPageProps> = ({
           exact: true,
         })
       ? TopologyViewType.graph
-      : getTopologyActiveView(activeViewStorageKey) || defaultViewType;
-    setQueryArgument('view', viewType);
+      : isTopologyViewStateLoaded && ((topologyViewState as TopologyViewType) || defaultViewType);
```

Comment:
Same comment about loaded.

Reply:
have added HOC for other two implementations and for this i think we are supporting backward compatibility so we  check path and proceed if exists otherwise use any stored preference. cc @jeff-phillips-18 

https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/topology/TopologyPage.tsx#L52-L63